node-wasm/src/error.rs

```diff
@@ -50,6 +50,16 @@ impl Error {
     }
 }
 
+impl Display for Error {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        if let Some(error) = self.0.dyn_ref::<js_sys::Error>() {
+            write!(f, "{} ({})", error.name(), error.message())
```

Comment:
could you also add a new line with the `.cause()`?

Reply:
Good idea, added printing `cause`s in a loop until we get to the bottom of the error 👍